src/arallocc.c

```diff
@@ -13,24 +21,17 @@
  * dynamically allocate internal C language arrays based on parameter
  * values set during one or more previous calls to function isetprm().
  *
- * <p>This subroutine isn't normally called directly from an application
+ * This subroutine isn't normally called directly from an application
  * program, since it's automatically called internally during the first
  * call to subroutine openbf() from an application program.
  *
- * @author J. Ator
- * @date 2014-12-04
- *
  * @remarks
- * - All memory allocated within this subroutine can be freed via a
- *   subsequent call to subroutine exitbufr() from within the
- *   application program, or else it will be freed automatically by the
- *   operating system once the application program terminates.
+ * All memory allocated within this subroutine can be freed via a
```

Comment:
FYI, the reason for the `-` character is so that remarks are rendered as bulletized in the HTML output.  Now granted, if there's only one remark (as in this routine), then it's not a big deal to omit this.  But if there are multiple remarks (as in many other routines) then I think it's very important that we distinguish them with `-` markup.

Reply:
We don't usually bulletize a remark and also mark it as a note.